doc/reference/configuration/configuration_reference.rst

```diff
@@ -3133,6 +3133,35 @@ log.syslog.*
     | Default: box.NULL
     | Environment variable: TT_LOG_SYSLOG_SERVER
 
+.. _configuration_reference_lua:
+
+lua
+---
+
+The ``lua`` section outlines the configuration parameters related to the ``lua`` environment within Tarantool.
+
+Below is an overview of the available configuration options for ``lua``:
+
+- :ref:`lua.memory <configuration_reference_lua>`
+
```

Comment:
Please check the [configuration scope](https://www.tarantool.io/en/doc/latest/platform/configuration/#configuration-scopes) to which the ``lua`` section applies.
I'll explain more in a call.